### ISSUE TITLE
Getting different no for source only experiments

### ISSUE THREAD

**Issue by Griffintaur** [zhengzangw/PCS-FUDA#2]
For experiments on the source-only value: I omit all the losses apart from the 'cls-so' for Product->Real experiment but I am getting usually high test accuracy of ~55% while the paper reported only 42%.
Should I do anything else apart from the above to get numbers for Source only experiments?

**Comment by zhengzangw** [comment]
I think one possible reason causing this is that in the current framework, even you delete all other losses, the target images can pass through the network, causing the BN layer to update. This will contribute to the final accuracy. In the source-only setting, target images should not be accessible to the network.

**Comment by Griffintaur** [comment]
I wrote a piece of code with all same hyper-parameters as PCS-FUDA for the training source only. Now I could achieve only 38 % accuracy as opposed to 42 % reported in the paper. (sent you an email on this)

**Comment by zhengzangw** [comment]
The source-only scenario is actually not implemented in the repo (for ease of implementation). As you find, BN only can raise the accuracy from 38% to 55% on your machine. We believe the few-shot and domain shift make the source-only training very unstable. Random seed, computing environment, or some minor implementation difference can result in the different accuracy of source-only scenarios.
